onnx/cppbuild.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# This file is meant to be included by the parent cppbuild.sh script
+if [[ -z "$PLATFORM" ]]; then
+    pushd ..
+    bash cppbuild.sh "$@" onnx
+    popd
+    exit
+fi
+
+if [[ $PLATFORM == windows* ]]; then
+    #No Windows support yet
+    :
+else
+
+    export PROTO=2.6.1
+    export INSTALL_PATH=`pwd`/$PLATFORM
+    wget https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh -O ~/miniconda.sh
+    bash ~/miniconda.sh -f -b -p $HOME/miniconda 
+    export PATH="$HOME/miniconda/bin:$PATH"
+    conda install -y -c conda-forge protobuf=2.6.1 numpy setuptools
```

Comment:
Does this need to create files in $HOME as hinted by the uninstallation procedure?
https://conda.io/docs/user-guide/install/linux.html#uninstalling-anaconda-or-miniconda
If we cannot get this to live somewhere else than $HOME, let's let users install it by themselves.
That is, place these commands in install-travis.sh for the CI servers as well as document them here:
https://github.com/bytedeco/javacpp-presets/wiki/Build-Environments#linux-x86-and-x86_64